codex/purchasing.nim

```diff
@@ -53,7 +53,7 @@ proc populate*(purchasing: Purchasing,
   if result.ask.proofProbability == 0.u256:
     result.ask.proofProbability = purchasing.proofProbability
   if result.expiry == 0.u256:
-    result.expiry = (purchasing.clock.now().u256 + purchasing.requestExpiryInterval)
+    result.expiry = purchasing.requestExpiryInterval
```

Comment:
Just a comment: `request.expiry` should never really be `0` as we checking for that in the rest api. I think this maybe isn't necessary at all, or actually we should be raising an error.

Reply:
You are right, but this is "older" than my original work on API side of the expiry. Mark, I guess originally envisioned the default value for expiry and hence this "populating" mechanism. I want to discuss it with him today to see if we can remove it in this PR...